backups/cognito/backup_users.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+import csv
+from datetime import datetime
+from botocore.exceptions import ClientError
+import os
+
+client = boto3.client('cognito-idp')
+s3_client = boto3.client('s3')
+
+def backup_users():
+    try:
+        all_users = get_all_users()
+    except Exception as e:
+        print('Getting current cognito users has failed ', e)
+
+    csv_headers = ['cognito:username','name','given_name',
+    'family_name','middle_name','nickname','preferred_username',
+    'profile','picture','website','email','email_verified',
+    'gender','birthdate','zoneinfo','locale','phone_number',
+    'phone_number_verified','address','updated_at','cognito:mfa_enabled']
```

Comment:
is there a way to avoid hardcoding them? Maybe a boto call that you can use to fetch all csv headers?

Reply:
then i would need to know where email, name and email_verified are so i can put them in the same index of the array I can create a loop with ifs to do that but do you think its worth it?